packages/ui/src/molecules/QuantitySelector/stories/QuantitySelector.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import type { Meta, Story } from '@storybook/react'
+import React, { useState, useMemo } from 'react'
+
+import type {
+  QuantitySelectorProps,
+  QuantitySelectorButtonProps,
+  QuantitySelectorInputProps,
+} from '..'
+import {
+  QuantitySelector,
+  QuantitySelectorButton,
+  QuantitySelectorInput,
+} from '..'
+import { PlusIcon, MinusIcon } from './assets/Icons'
+import mdx from './QuantitySelector.mdx'
+
+const MAX_QUANTITY = 10
+const MIN_QUANTITY = 1
+
+type QuantitySelectorCompoundProps = QuantitySelectorProps &
+  QuantitySelectorButtonProps &
+  QuantitySelectorInputProps
+const QuantitySelectorStylelessTemplate: Story<QuantitySelectorCompoundProps> = (
+  args
+) => {
+  const [quantity, setQuantity] = useState(MIN_QUANTITY)
+
+  function isLeftDisabled() {
+    return quantity === MIN_QUANTITY
+  }
+
+  function isRightDisabled() {
+    return quantity === MAX_QUANTITY
+  }
+
+  return (
+    <QuantitySelector
+      {...args}
+      name="quantity-selector"
+      currentValue={quantity}
+      onClick={(v) => {
+        if (v.currentTarget.name === 'quantity-selector-increase-button') {
+          setQuantity((currentQuantity) =>
+            Math.min(currentQuantity + 1, MAX_QUANTITY)
+          )
+        } else if (
+          v.currentTarget.name === 'quantity-selector-decrease-button'
+        ) {
+          setQuantity((currentQuantity) =>
+            Math.max(currentQuantity - 1, MIN_QUANTITY)
+          )
+        }
+      }}
+    >
+      <QuantitySelectorButton
```

Comment:
It's also missing labels for these buttons (`QuantitySelectorButton`)
I'd suggest adding `aria-label="Decrement"` here.



https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

Reply:
Btw, I'm not sure if I mentioned this before, but you can use the Accessibility addon panel to help you and get more information about it. 

![image](https://user-images.githubusercontent.com/3356699/144885223-8fcce188-0615-4f6d-b191-ff132d854e35.png)

Reply:
This Accessibility tab does help a lot. It wasn't showing up at my local environment, but after a browser cache clearing, it's working. I believe https://github.com/vtex/faststore/pull/1035/commits/dfaaee45a47e61f6968277bc4f215e5c6a78bd9c solves this.